Release oasis storage prune and compact subcomands (#6521)
oasisprotocol/oasis-core: docs/oasis-node/cli.md

## storage

### compact

> Run (when the node is not running):

Reviewer: Can it automatically check whether the node is not running before compacting the storage? Same for the prune below.

Author: Already the case for both commands, see `PreRunE`.

way they guarantee the node is healthy when it starts.

> Following successful pruning, to release disk space, they are encouraged to run

Reviewer: So many words :)

```suggestion
After the pruning operators should run [the compaction command](#compact) to release disk space.
```
